conf/main/logback.xml

```diff
@@ -62,11 +62,11 @@
         <appender-ref ref="${jetty.log.appender:-MAIN}"/>
     </logger>
 
-    <logger name="ai.grakn" level="${grakn.log.level}">
+    <logger name="ai.grakn" level="${grakn.log.level:-INFO}">
```

Comment:
I personally find that confusing. I would say it should be `-LOG` if it is going to contain all log levels. The `-INFO` suffix indicates we may be dividing our logs into different levels when we actually not.